src/common/Utilities/StringFormat.cpp

```diff
@@ -37,5 +37,19 @@ Str Acore::String::Trim(const Str& s, const std::locale& loc /*= std::locale()*/
     return s;
 }
 
+std::string Acore::String::TrimRightInPlace(std::string& str)
+{
+    int pos = int(str.size()) - 1;
```

Comment:
dont you need to specify the integer? like int32 or smt?

Reply:
This just copy from poco lib

Reply:
👍